aom/src/main/java/com/nedap/archie/rules/Operator.java

```diff
@@ -14,6 +14,9 @@
 
     private List<Expression> operands = new ArrayList<>();
 
+    private String symbol;
+
+    @JsonIgnore
     public OperatorKind getOperator() {
         return operator;
     }
```

Comment:
Now that this field is ignored in StandardsCompliant and ForJavascript, do we still need the OldOperatorKindSerializer and all the annotations in OperatorKind?

Reply:
Yes. It is still used in OperatorLegacyFormatMixin. We could remove the op_ serialiation because it's only used in legacy mode, but I think having the standards by default in the enum is better.